spec/lib/triannon-client/triannon-client_spec.rb

```diff
@@ -91,86 +177,127 @@ def check_graph_has_statements(graph)
     end
     context 'with no content_type' do
       it 'raises an argument error with a nil ID' do
-        expect{@tc.get_annotation(nil)}.to raise_error(ArgumentError)
+        tc = TriannonClient::TriannonClient.new
```

Comment:
if you're going to do this line a lot, you might consider DRYing up the code and putting it in a before(:each) or let statement for the context block